test/optimizations/arrayViewElision/remoteDR.hcpl

```diff
@@ -0,0 +1 @@
+var A: 
```

Comment:
I don't think you meant to add this file

Reply:
I am not going to admit how long it took me to understand what's going on here.